scripts/binary-release.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
This used to exist in upstream but they removed it in favour of pushing release artifacts with github actions. I added it back so we can keep building them ourselves for our fork.